dashboard/app/models/contact_rollups_final.rb

```diff
@@ -23,6 +23,6 @@ def self.insert_from_processed_table
       FROM contact_rollups_processed;
     SQL
 
-    ActiveRecord::Base.connection.exec_query(insert_sql)
+    transaction {ActiveRecord::Base.connection.exec_query(insert_sql)}
```

Comment:
Is this just for consistency with other places where we're using transactions? Or is there actually a benefit of wrapping this in a transaction?

Reply:
There is one benefit, `ContactRollupsFinal` will be empty if this fails, thus there is no risk of accidentally using incomplete data.